package.json

```diff
@@ -19,13 +19,14 @@
     "prepare": "npm run build"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "^1.113.0",
+    "@aws-cdk/assert": "^1.128.0",
     "@commitlint/cli": "^12.1.4",
     "@commitlint/config-conventional": "^12.1.4",
     "@types/jest": "^26.0.10",
     "@types/node": "10.17.27",
     "@typescript-eslint/eslint-plugin": "^4.28.3",
     "@typescript-eslint/parser": "^4.28.3",
+    "esbuild": "^0.13.8",
```

Comment:
Used by https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-lambda-nodejs

Reply:
It seems to be a `devDependency` for that project, though, not a `peerDependency`. Why do we need to add it here?

Reply:
Because if `esbuild` isn't installed then a `cdk synth` will try and run `esbuild` inside a docker container. This works fine until you start running `cdk synth` inside a docker container (like we do in CircleCI). It is also significantly quicker doing a `cdk synth` with `esbuild` already installed.